src/App.js

```diff
@@ -28,9 +28,23 @@ const App = (props) => {
     const [arePermissionsReady, setArePermissionReady] = useState(false);
     const [hasSystems, setHasSystems] = useState();
 
-    async function initChrome () {
+    async function initChrome() {
         insights.chrome.init();
         insights.chrome.identifyApp('dashboard');
+
+        insights.chrome?.globalFilterScope?.('insights');
+        if (insights.chrome?.globalFilterScope) {
+            insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
+                const selectedTags = insights.chrome?.mapGlobalFilter?.(data)?.filter(item => !item.includes('Workloads')) || undefined;
+                const encodedTags = selectedTags.map(tag => {
```

Comment:
Aren't you doing effectively the same thing twise? The mapGlobalFilter function returns an array of selected tags, you split them and encode them. Instead of this we should encode the tags in chrome function `mapGlobalFilter`

Reply:
depends, some apis need it, maybe others dont? would be great if we didn't have to do this, but presently mapGlobalFilter doesn't encode the namespace/key/value

Reply:
I'm fine by encoding these things in chrome and if any app wants unencoded partials, they can always write their own mapper. Yeah, I figured that encoding full tag would look strange.

Reply:
AWESOME! sweet! ok so if nothing else looks askew here, will merge this one, n remove the mapping encoder thingy once that shows up 🙇 🤗